### ISSUE TITLE
Traceback on extra bindings since v5.1

### ISSUE THREAD

**Issue by j-cote100** [ragardner/tksheet#91]
I just updated tksheet from 5.0.34 to 5.1.16 and there seem to be an issue with my extra bindings.  

the below example code works perfectly on v5.0.34 but I get an error on v5.1.16 (Actually anything on v5.1.x)

```
from tksheet import Sheet
import tkinter as tk

data = [[1, 2, 3, 4, 5, 6],
        [1, 2, 3, 4, 5, 6],
        [1, 2, 3, 4, 5, 6],
        [1, 2, 3, 4, 5, 6]]

class Display(tk.Toplevel):
    def __init__(self, parent):
        self.parent = parent
        tk.Toplevel.__init__(self, self.parent)

        self.frame = tk.Frame(self)
        self.frame.grid_columnconfigure(0, weight=1)
        self.frame.grid_rowconfigure(0, weight=1)

        self.sheet = Sheet(self.frame, data=data)
        self.sheet.set_all_column_widths()

        self.frame.grid(row=0, column=0, sticky="nsew", rowspan=20)
        self.sheet.grid(row=0, column=0, sticky="nswe", rowspan=20)
        self.add_bindings()

    def add_bindings(self):
        self.sheet.enable_bindings(("single_select",
                                    "column_select",
                                    'row_select',
                                    "right_click_popup_menu",
                                    "rc_select",
                                    "rc_insert_row",
                                    "rc_delete_row",
                                    "arrowkeys",
                                    "undo",
                                    "edit_cell"))

        self.sheet.extra_bindings('begin_edit_cell', func=self.begin_edit_cell)
        self.sheet.extra_bindings('end_edit_cell', func=self.end_edit_cell)

    def begin_edit_cell(self, event):
        print(event)

    def end_edit_cell(self, event):
        print(event)


root = tk.Tk()
Display(root)
root.mainloop()
```

Here's the traceback that's raised when I simply click on any cell in the sheet

```
Traceback (most recent call last):
  File "~\lib\tkinter\__init__.py", line 1892, in __call__
    return self.func(*args)
  File "~\lib\site-packages\tksheet\_tksheet_main_table.py", line 2444, in b1_release
    self.hide_dropdown_window()
  File "~\lib\site-packages\tksheet\_tksheet_main_table.py", line 5057, in hide_dropdown_window
    self.destroy_text_editor("Escape")
  File "~\lib\site-packages\tksheet\_tksheet_main_table.py", line 4779, in destroy_text_editor
    self.extra_end_edit_cell_func(self.text_editor_loc + ("Escape", None))
	
TypeError: unsupported operand type(s) for +: 'NoneType' and 'tuple'
```

**Comment by ragardner** [comment]
Sorry about that, it should be fixed with `5.2.0` but please note that if you have an extra binding for begin edit cell, if the function has a return value which is not `None` then the text editor window will be populated with it

**Comment by j-cote100** [comment]
Just tested it out with 5.2.0 and it works fine now.  Noted for the return value in the begin edit cell function. 
Thank you very much
